specification/netapp/resource-manager/Microsoft.NetApp/stable/2022-01-01/netapp.json

```diff
@@ -6646,18 +6640,9 @@
       "type": "object",
       "x-ms-azure-resource": true,
       "required": [
-        "location",
         "properties"
       ],
```

Comment:
Are you updating a tracked resource to proxy or was it proxy always and you are just removing the location property because it was added by mistake ?

Reply:
It was always a proxy, tags where added by mistake.